pkg/smart/common.go

```diff
@@ -62,14 +59,39 @@ func detectSCSIType(name string) (Dev, error) {
 	return &device, nil
 }
 
-// readDeviceCapacity sends a SCSI READ CAPACITY(10) command to a device and
-// returns the capacity in bytes. If this command fails due to a capacity
-// overflow, it retries a SCSI READ CAPACITY(16) command.
+// isReadDeviceCapacity16Supported checks if the SCSI READ CAPACITY(16)
+// command is supported by the device.
+func (d *SCSIDev) isReadDeviceCapacity16Supported() (bool, error) {
+	respBuf := make([]byte, 20)
+
+	cdb := CDB12{}
+	cdb[0] = SCSIReportSupportedOpCodes
+	cdb[1] = SCSIReportSupportedOpCodesSA
+	cdb[2] = 0x02
```

Comment:
Can we use 0x02 as a constant and may be have some description (or the constant could itself be descriptive) why we need to populate this particular field ?

Reply:
No issue @patheticpat. It's ok if we even give some description for the field instead of having the constant.